cocotb/decorators.py

```diff
@@ -567,6 +570,9 @@ async def f(*args, **kwargs):
         self.timeout_unit = timeout_unit
         self.expect_fail = expect_fail
         if expect_error is True:
+            warnings.warn(
+                "Passing `expect_error=True` to `cocotb.test` is deprecated, specify a specific Exception type instead",
```

Comment:
```suggestion
                "Passing `expect_error=True` to `cocotb.test` is deprecated, pass a specific Exception type instead",
```
Using "specific" twice here is odd.